include/mrbconf.h

```diff
@@ -38,7 +38,12 @@
 
 /* add -DMRB_METHOD_T_STRUCT on machines that use higher bits of pointers */
 /* no MRB_METHOD_T_STRUCT requires highest 2 bits of function pointers to be zero */
-//#define MRB_METHOD_T_STRUCT
+#ifndef MRB_METHOD_T_STRUCT
+  // can't use highest 2 bits of function pointers on 32bit Windows.
+# if defined(_WIN32) && defined(MRB_32BIT)
+#   define MRB_METHOD_T_STRUCT
+# endif
+#endif
```

Comment:
`_WIN32` also seems to be defined on 64-bit Windows.

https://docs.microsoft.com/en-US/cpp/preprocessor/predefined-macros?view=vs-2019

> _WIN32 Defined as 1 when the compilation target is 32-bit ARM, 64-bit ARM, x86, or x64. Otherwise, undefined.
>
> _WIN64 Defined as 1 when the compilation target is 64-bit ARM or x64. Otherwise, undefined.

I think that the condition should be modified if only 32-bit Windows is targeted.

Reply:
Should we use `defined(_WIN32) && !defined(_WIN64)` instead of `defined(_WIN32) && defined(MRB_32BIT)` to detect 32bit Windows?

Reply:
As far as I can see from the reference above, I think that's fine (Unfortunately I have no further information).

Reply:
OK.
I've changed.